app/src/data_access_layer/repositories/session.repository.ts

```diff
@@ -124,6 +121,18 @@ export class SessionRepository {
     }
   }
 
+  public async findAll(): Promise<Session[]> {
+    try {
+      const sessions: Session[] = await this.databaseClient.session.findMany();
+      if (!sessions) throw new DatabaseError("Sessions don't exist");
```

Comment:
Throwing an error seems like the wrong approach to me. empty array means there are no sessions recorded yet, it shouldn't be an error